src/pages/Swap/index.tsx

```diff
@@ -838,14 +833,8 @@ export function Swap({
           )}
         </div>
       </AutoColumn>
-      {!showOptInSmall && !isUniswapXDefaultEnabled && <UniswapXOptIn isSmall={false} swapInfo={swapInfo} />}
     </SwapWrapper>
   )
 
-  return (
-    <>
-      {swapElement}
-      {showOptInSmall && !isUniswapXDefaultEnabled && <UniswapXOptIn isSmall swapInfo={swapInfo} />}
-    </>
-  )
+  return <>{swapElement}</>
```

Comment:
i think you can now return the swapElement directly instead of shoving it in a variable